src/client/sky.cpp

```diff
@@ -809,7 +804,6 @@ void Sky::setMoonTexture(const std::string &moon_texture,
 		m_materials[4].setTexture(0, m_moon_texture);
 		m_materials[4].MaterialType = video::EMT_TRANSPARENT_ALPHA_CHANNEL;
 		disableTextureFiltering(m_materials[4]);
-		m_materials[4].Lighting = !!m_moon_tonemap;
```

Comment:
The main thing left to do for this PR is to figure out what this and the rest of sky.cpp were doing with that lighting